draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1041,20 +1080,21 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4 - W3C Verifiable Credentials Data Model (work in progress)
+## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
 
 This example illustrates how the artifacts defined in this specification can be
-represented using W3C Verifiable Credentials Data Model as defined in
-[@VC_DATA].
+represented as W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] without using JSON-LD.
```

Comment:
again "with the credential content being a set of JSON claims" "with JSON based content".

Reply:
JSON-LD is JSON, too. We say `without using JSON-LD` in OID4VP spec. I want to be explicit.